test/Cnblogs.Architecture.IntegrationTests/DaprTests.cs

```diff
@@ -34,7 +34,27 @@ public async Task Dapr_SubscribeEndpoint_OkAsync()
         // Assert
         response.StatusCode.Should().Be(HttpStatusCode.OK);
         var responseText = await response.Content.ReadAsStringAsync();
-        responseText.Should().Contain("pubsub");
+        responseText.Should().Contain(nameof(TestIntegrationEvent));
+    }
+
+    [Fact]
+    public async Task Dapr_Subscribe_Without_Any_Assembly_OkAsync()
```

Comment:
```suggestion
    public async Task Dapr_SubscribeWithoutAnyAssembly_OkAsync()
```

参考：https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

它不是 underscore case，只是用下划线区分 `项目_条件_结果`，其实是最多只有两个下划线的。